ravenframework/Optimizers/GeneticAlgorithm.py

```diff
@@ -538,25 +557,6 @@ def _useRealization(self, info, rlz):
         children = self._repairInstance(childrenMutated,variables=list(self.toBeSampled),distInfo=self.distDict)
       else:
         children = childrenMutated
-      # Make sure no children are exactly similar to parents
-      flag = True
-      counter = 0
-      while flag and counter < self._populationSize:
-        counter += 1
-        repeated = []
-        for i in range(np.shape(self.population.data)[0]):
-          for j in range(i,np.shape(children.data)[0]):
-            if all(self.population.data[i,:]==children.data[j,:]):
-              repeated.append(j)
-        repeated = list(set(repeated))
-        if repeated:
-          if len(repeated)> children.shape[0] - self._populationSize:
-            newChildren = self._mutationInstance(offSprings=children[repeated,:], distDict=self.distDict, locs=self._mutationLocs, mutationProb=self._mutationProb, variables=list(self.toBeSampled))
-            children.data[repeated,:] = newChildren.data
-          else:
-            children = children.drop_sel(chromosome=repeated)
-        else:
-          flag = False
```

Comment:
This portion of code has been commented out, but its functionalities have not been replaced. is it intentional?

Reply:
This was to prevent the repetition of chromosomes which delays convergence, I have been wanting to remove it for a while but was a little coughtous about losing diversity promoting mechanisms and getting stuck in local optima. Now we are trying to test that. This will keep the elite parents.

Reply:
as mentioned in the meeting, I'd suggest to have the user specify a number of initial iteration where this code is used to increase population diversification.